tests/utils/create-e2e-fixture.ts

```diff
@@ -248,7 +256,7 @@ async function installRuntime(
       env['YARN_ENABLE_SCRIPTS'] = 'false'
       break
     case 'pnpm':
-      command = `pnpm add file:${join(isolatedFixtureRoot, packageName)} ${
+      command = `pnpm add file:${relativePathToPackage} ${
```

Comment:
relative path needed for buildbot to actually find tarballed runtime in zip file - this possibly will have to be replicated for other package managers if we will need to use buildbot for those (I only did `pnpm` here, `npm` already worked when buildbot builds were added for skew protection)